hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/TestSqlFileBasedSource.java

```diff
@@ -63,7 +63,7 @@ public class TestSqlFileBasedSource extends UtilitiesTestBase {
 
   @BeforeAll
   public static void initClass() throws Exception {
-    UtilitiesTestBase.initTestServices(true, true, false);
+    UtilitiesTestBase.initTestServices(false, true, false);
```

Comment:
Are these all the places we use HDFS?

Reply:
Tests that uses utilitiesTestBase. I will keep removing more.